community/libmagick6/build

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh -e
+
+./configure \
+    --prefix=/usr \
+    --sysconfdir=/etc \
+    --with-modules
+
+sed -i -e 's/ -shared / -Wl,-O1,--as-needed\0/g' libtool
```

Comment:
Reason for this change?

Reply:
monkey read arch pkgbuild monkey do. 

imagemagick was complex to build. 

maybe we don't need this section after all?

Reply:
How about we remove and see how long it lasts before we need it

Reply:
My `imagemagick7` package in my personal repository is a simple `./configure; make; make install`. Nothing special needed. I say remove it.

Another question, why not ImageMagick7 (the latest version)?

Reply:
I need imagemagick6 for Emacs, and it is still maintained upstream so it's fine